### PR TITLE
Added pygments as it is needed by demo/showcase

### PR DESCRIPTION
**Pull request by ypwangnexus**
Confirmed and tested on 
macOS Sierra 10.12.6
Xcode 9.2
Simulator iPhone 8 Plus - 11.2

### CODE REVIEW

File: recipes/pygments/__init__.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# pure-python package, this can be removed when we'll support any python package
+from toolchain import PythonRecipe
+
+
+class PlyerRecipe(PythonRecipe):
```
Comment (KeyWeeUsr) [inline recipes/pygments/__init__.py]: Shouldn't this be `PygmentsRecipe` instead? :)

File: recipes/pygments/__init__.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+    version = "2.2.0"
+    url = "https://pypi.python.org/packages/71/2a/2e4e77803a8bd6408a2903340ac498cb0a2181811af7c9ec92cb70b0308a/Pygments-2.2.0.tar.gz"
+    depends = ["python"]
+    archs = ["i386"]
+
+recipe = PlyerRecipe()
```
Comment (KeyWeeUsr) [inline recipes/pygments/__init__.py]: same

File: recipes/pygments/__init__.py (hunk 1 above)
Comment (ypwangnexus) [reply recipes/pygments/__init__.py]: Yes:-)

File: recipes/pygments/__init__.py (hunk 2 above)
Comment (ypwangnexus) [reply recipes/pygments/__init__.py]: Say yes

**Pull request closed without merging**